ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```
@@ -749,6 +749,160 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )edge-e2e-ai-operator-ztp-ipv4v6-sno-ocp-49,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

```suggestion
    always_run: false
```
otherwise it will start running (while not being required) for all PRs

---

Fixed